com.unity.multiplayer.mlapi/Runtime/Spawning/NetworkSpawnManager.cs

```diff
@@ -68,11 +68,15 @@ public NetworkObject GetLocalPlayerObject()
         }
 
         /// <summary>
-        /// Returns the player object with a given clientId or null if one does not exist
+        /// Returns the player object with a given clientId or null if one does not exist. This is only valid server side.
         /// </summary>
         /// <returns>The player object with a given clientId or null if one does not exist</returns>
         public NetworkObject GetPlayerNetworkObject(ulong clientId)
         {
+            if (!NetworkManager.Singleton.IsServer && NetworkManager.Singleton.LocalClientId != clientId)
```

Comment:
This will still not work for a client/host to get their local player player object. See #580.  I think if we fix this we should fix this properly. @TwoTenPvP recommended to just add the local client to the ConnectedClients dictionary I think.

Reply:
It won't throw an exception if local id == clientId, so that should still work no?
I didn't want to actually include other clients in there per my PR comment. Do you think security is not a concern here?

Reply:
To clarify. Right now as a client I cannot retrieve *any* player object. Not even my own one. See issue #580

Reply:
The issue I mentioned has been fixed in #706. This change should be good enough